src/Uno.UWPSyncGenerator/DocGenerator.cs

```diff
@@ -222,6 +224,65 @@ private bool IsViewType(INamedTypeSymbol type)
 			return IsViewType(type.BaseType);
 		}
 
+		private string ConstructBaseClassString(PlatformSymbols<INamedTypeSymbol> view)
+		{
+			var uniqueBaseTypes = AllSymbols()
+				.Where(s => s.Symbol != null)
+				.Select(s => (s.Symbol.BaseType.Name, s.Symbol.BaseType.ContainingNamespace.ToDisplayString()))
+				.Distinct()
+				.ToArray();
+			if (uniqueBaseTypes.Length == 1)
+			{
+				var tuple = uniqueBaseTypes.Single();
+				return MarkdownStringBuilder.Hyperlink(tuple.Item1, GetLinkTarget(tuple));
+			}
+			else
+			{
+				var output = "";
+				foreach (var tuple in uniqueBaseTypes)
+				{
+					var link = GetLinkTarget(tuple);
+					var name = link != null ?
+						MarkdownStringBuilder.Hyperlink(tuple.Item1, link) :
+						tuple.Item1;
+					var matchingPlatforms = AllSymbols()
+						.Where(s => s.Symbol != null)
+						.Where(s => s.Symbol.BaseType.Name == tuple.Item1 && s.Symbol.BaseType.ContainingNamespace.ToDisplayString() == tuple.Item2)
+						.Select(t => ToDisplayString(t.ImplementedFor));
+					output += $"{name} ({matchingPlatforms.JoinBy(@"/")}) ";
+				}
+
+				return output;
+			}
+
+			string GetLinkTarget((string symbolName, string symbolNamespace) tuple)
+			{
+				if (tuple.symbolNamespace == view.UAPSymbol.ContainingNamespace.ToDisplayString())
+				{
+					return $"#{tuple.symbolName}";
+				}
+				else if (_kosherFrameworkViews.Contains(tuple))
+				{
+					return $"{GetImplementedMembersFilename(tuple.symbolNamespace)}#{tuple.symbolName}";
+				}
+				else
+				{
+					return null;
+				}
+			}
+
+			IEnumerable<(INamedTypeSymbol Symbol, ImplementedFor ImplementedFor)> AllSymbols()
+			{
+				yield return (view.UAPSymbol, ImplementedFor.UAP);
+				yield return (view.AndroidSymbol, ImplementedFor.Android);
+				yield return (view.IOSSymbol, ImplementedFor.iOS);
+				yield return (view.WasmSymbol, ImplementedFor.WASM);
+#if HAS_MACOS
```

Comment:
You can take this out

Reply:
Done, and I've adjusted the output to include MacOS